New Weapon Models

Hathoom opened the pull request:
Added a stapler, rubberband, staples, and 2 sticks to be put on the Model.

Hathoom committed:
New Weapon Models

Added a stapler, rubberband, staples, and 2 sticks to be put on the Model.

folsterA commented:
Why are a bunch of assets from Brendon's assets deleted? It doesn't say in the commits, and I can't find their file names in other asset folders although I may be blind.

Hathoom committed:
Updates

Removed Brendon's Flo.meta, I don't know what it does, but it isn't on main and removing it here didn't have Unity scream at me.

SF_ShootingScene had a missing prefab which was throwing up an error.  I removed the missing prefab and replaced it with a new one, but I didn't hook up the scene at all.
